app/code/core/Mage/Customer/controllers/AccountController.php

```diff
@@ -184,6 +184,7 @@ public function loginPostAction()
      */
     protected function _loginPostRedirect()
     {
+        Mage::dispatchEvent('before_login_post_redirect', ['controller' => $this]);
```

Comment:
thinking about it, the name of this event is really unclear and since it wasn't working before... can't we just use a better name? should this also be added to the events list?